src/Features/Cipher/Autokey/autokeyCipherSlice.ts

```diff
@@ -0,0 +1,43 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export interface State {
+  conversion: number;
+  secret: string;
+  text: string;
+}
+
+const initialState: State = {
+  conversion: 2,
+  secret: '',
+  text: '',
+};
+
+const autokeyCipherSlice = createSlice({
+  name: 'autokeyCipher',
+  initialState,
+  reducers: {
+    clear(state) {
+      state.conversion = 2;
```

Comment:
Could this be either

state = initialState
Or
state.converstion = initialState.converstion // etc.